### ISSUE TITLE
Underestiamtion of heat load 

### ISSUE THREAD

**Issue by PRemmen** [RWTH-EBC/TEASER#461]
also reported by @mschumacher247 

## What is the problem?

- It seems, that especially for well insulated buildings, like TABULA retrofit and advanced retrofit, TEASER underestimates the static heat load of the building.
- This leeds to maximum power use of the simple heater in the simulation and quite often the set temperature is violated, because the given power is to low

## Why do we want to solve it?

- Because this is in unintended behaviour of our model

## How do we want to solve it?

- one option would be to have some safety value, e.g. add 20% on top of the calculated static heat load
- any other ideas?

**Comment by mlauster** [comment]
Good question why we underestimate that... . Maybe we should check the implementation of heat load calculation and see, if the infiltration rate is set too low and if we violate this range when simulating because of higher infiltration rates at the minimum temperatures...

**Comment by mla-srs** [comment]
The infiltration rate is set as 0.5.
For a multi family residential building or a non residential building (not industrial) it is defined in EN15242 for a difference pressure of 4 Pa as 0.4 1/h and 10 Pa as 0.8 1/h for medium leakage. 

@mlauster The pressure difference should get bigger for bigger temperature differences. Which would lead to a higher infiltration rate i.e. 50Pa -> 2 1/h. Is that what you meant?

**Comment by mlauster** [comment]
@mla-srs, yes, this is what I meant! Before adding a rough perecentage to be on the safer side, we should check if it helps to raise the infiltration rate while still staying in reasonable ranges. I would propose using 0.8 or 1.

**Comment by mla-srs** [comment]
@PRemmen What do you think of @mlauster's suggestion?
A infiltration rate in dependence of the temperature difference is too complex isn't it?

**Comment by PRemmen** [comment]
@mla-srs Mhh not quite sure, because the error seems to occur especially for new, well insulated buildings, where there is typically a leakage rate.

We can do this as this could be something like a "safety value" however this should be documented very well

**Comment by mschumacher247** [comment]
I added a first draft for a improved heat load calculation. However following DIN EN 12831 there are some dependencies/assumptions that need to be solved. Most important are:
- [ ] Building mass class for re-heating factors
- [ ] Method of re-heating factor calculation (temp drop vs. non-usage time)
